src/ai_marketplace_monitor/facebook.py

```diff
@@ -529,7 +529,19 @@ def search(
                         self.logger.debug(
                             f"""{hilight("[Retrieve]", "succ")} New item "{listing.title}" from https://www.facebook.com{listing.post_url} is sold by "{listing.seller}" and with description "{listing.description[:100]}..." """
                         )
-                    if self.check_listing(listing, item_config):
+
+                    # Warn if we never managed to extract a description for keyword-based filtering
+                    if (
+                        (not listing.description or len(listing.description.strip()) == 0)
+                        and item_config.keywords
+                        and len(item_config.keywords) > 0
+                    ):
+                        if self.logger:
+                            self.logger.error(
```

Comment:
Description is optional for facebook listings, so you may get false warning for missing description. Perhaps you can remove the warning or change it to `debug`?